app/views/declare-partnership/single/failed.html

```diff
@@ -34,6 +27,13 @@ <h1 class="govuk-heading-xl">
                     }
                 ]
             }) }}
+            <h1 class="govuk-heading-xl">
+                Add new partnerships
+            </h1>
+            {{ govukWarningText({
+                            text: "You should only add a school to your list if they have signed a partnership contract with you",
+                            iconFallbackText: "Warning"
+                        }) }}
```

Comment:
Whitespace isn't quite right.

Also, are we sure about moving the page header below error summary?

Reply:
The error summary thing was a specific request from Dom

Reply:
Ah yeah, good spot on the whitesapce. 

We had a chat with Dom yesterday about the error summary and she asked to put the page header below the error summary. There's a screenshot on the miro board how the coronavirus service has it.

Reply:
Ah, alright, I should have read https://design-system.service.gov.uk/components/error-summary/ in more detail.
> Put the error summary at the top of the main container. If your page includes breadcrumbs or a back link, place it below these, but above the \<h1>.